pkg/api/v1/routes/handlers.go

```diff
@@ -270,10 +345,14 @@ func (r *Routes) serverConditionDelete(c *gin.Context) {
 			http.StatusBadRequest,
 			&v1types.ServerResponse{Message: "unsupported condition kind: " + string(kind)},
 		)
+		r.logger.WithFields(logrus.Fields{
+			"kind": kind,
+		}).Info("unsupported condition kind")
 
 		return
 	}
 
+	// TODO (vc): the repository Delete route should take a condition id
```

Comment:
More of a discussion topic we can have separately, where do we persist the condition IDs.